ProjectFiles/Assets/Scripts/Network/EventType.cs

```diff
@@ -5,17 +5,21 @@ namespace Network
     [Flags]
     public enum EventType
     {
-        Undefined = 0xFF,
-        ClientHandshake = 0x81,
-        ClientLocationUpdate = 0x82,
-        ClientSpaceEnterEvent = 0x83,
-        ClientSpaceExitEvent = 0x84,
-        ServerHandshake = 0x01,
-        ServerLocationUpdate = 0x02,
-        ServerSpawnPlayerEvent = 0x03,
-        ServerPreRoundStartEvent = 0x04,
-        ServerRoundStartEvent = 0x05,
-        ServerRoundEndEvent = 0x06,
+        Undefined                   = 0xFF,
+        
+        ClientHandshake             = 0x81,
+        ClientLocationUpdate        = 0x82,
+        ClientSpaceEnterEvent       = 0x83,
+        ClientSpaceExitEvent        = 0x84,
+        
+        AdminClientStartGameEvent   = 0xA0,
+        
+        ServerHandshake             = 0x01,
+        ServerLocationUpdate        = 0x02,
+        ServerSpawnPlayerEvent      = 0x03,
+        ServerPreRoundStartEvent    = 0x04,
+        ServerRoundStartEvent       = 0x05,
+        ServerRoundEndEvent         = 0x06,
```

Comment:
This aligned stuff will be gone next time I press 'reformat' ;p

Reply:
And itll go back when I next use rider :smile: 

https://blog.jetbrains.com/dotnet/2017/11/28/aligning-code-columns-resharper-rider/